emcl2_node:
  ros__parameters:
    odom_freq: 20
    global_frame_id: "map"
    footprint_frame_id: "base_footprint"
    odom_frame_id: "odom"
    base_frame_id: "base_link"
    laser_min_range: 0.0
    laser_max_range: 100000000.0
    scan_increment: 1
    initial_pose_x: 0.0
    initial_pose_y: 0.0
    initial_pose_a: 0.0
    num_particles: 0
    alpha_threshold: 0.5
    expansion_radius_position: 0.1 
    expansion_radius_orientation: 0.2
    extraction_rate: 0.1
    range_threshold: 0.1
    sensor_reset: true